cmd/ctl/pkg/convert/convert.go

```diff
@@ -68,33 +61,21 @@ to change to output destination.`))
 )
 
 var (
-	scheme             = runtime.NewScheme()
-	codecs             = serializer.NewCodecFactory(scheme)
-	parameterCodec     = runtime.NewParameterCodec(scheme)
-	localSchemeBuilder = runtime.SchemeBuilder{
-		cmapi.AddToScheme,
-		cmapiv1alpha3.AddToScheme,
-		cmacme.AddToScheme,
-		cmacmev1alpha3.AddToScheme,
-		cmmeta.AddToScheme,
-		whapi.AddToScheme,
-		kscheme.AddToScheme,
-		apireg.AddToScheme,
-		apiext.AddToScheme,
-		auditreg.AddToScheme,
-	}
-
-	addToScheme = localSchemeBuilder.AddToScheme
+	// Use the webhook's scheme as it already has the internal cert-manager types,
+	// and their conversion functions registered.
+	// In future we may we want to consider creating a dedicated scheme used by
+	// the ctl tool.
+	scheme = webhook.Scheme
 )
 
 func init() {
-	// This is used to add the List object type for outputing multiple input
+	// This is used to add the List object type for outputting multiple input
 	// objects.
 	coreGroupVersion := schema.GroupVersion{Group: "", Version: runtime.APIVersionInternal}
 	scheme.AddKnownTypes(coreGroupVersion, &metainternalversion.List{})
 
 	metav1.AddToGroupVersion(scheme, schema.GroupVersion{Version: "v1"})
-	utilruntime.Must(addToScheme(scheme))
+	utilruntime.Must(kscheme.AddToScheme(scheme))
 }
```

Comment:
This whole `func init()` function can be removed as far as I'm aware? All of these things should be registered (if required) into the webhook scheme already 😄